terraform/bastion/firewall.tf

```diff
@@ -1,40 +1,10 @@
 // The resources in this file control who has access to the bastion server.
 
-locals {
-  // Users allowed to connect to the bastion through SSH. Each user needs to
-  // have the CIDR of the static IP they want to connect from stored in AWS SSM
-  // Parameter Store (us-west-1), in a string key named:
-  //
-  //     /prod/bastion/allowed-ips/${user}
-  //
-  allowed_users = [
-    "aidanhs",
-    "joshua",
-    "pietro",
-    "shep",
-    "simulacrum",
-    "technetos",
-    "nemo157",
-    "syphar",
-    "rylev",
-    "rylev-ip-2",
-    "jdn",
-    "guillaumegomez",
-    "marcoieni",
-  ]
-}
```

Comment:
Is this the list of allowed users now?
https://github.com/rust-lang/simpleinfra/blob/ca5e95c263f4fe4392fb4de4160f21bf0551f42c/ansible/playbooks/bastion.yml#L13